src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,20 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot;
+
+/**
+ * Add your docs here.
+ */
+public class Constants {
+
+    public Constants() {
```

Comment:
You don't need an empty constructor in a class. The compiler will generate one if it doesn't exist.

Reply:
Removed.  Though I don't like depending on the automatic constructor.  Guess it's the c++ developer in me.